src/kixi/datastore/metadatastore/license.clj

```diff
@@ -0,0 +1,31 @@
+(ns kixi.datastore.metadatastore.license
+  (:require [clojure.spec :as s]))
+
+(s/def ::usage string?)
+
+(def licenses
+  {"Creative Commons Attribution" {}
+   "Creative Commons Attribution Share-Alike" {}
+   "Creative Commons CCZero" {}
+   "Creative Commons Non-Commercial (Any)" {}
+   "GNU Free Documentation License" {}
+   "Not Specified" {}
+   "Open Data Commons Attribution License" {}
+   "Open Data Commons Open Database License (ODbL)" {}
+   "Open Data Commons Public Domain Dedication and License (PDDL)" {}
+   "Other (Attribution)" {}
+   "Other (Non-Commercial)" {}
+   "Other (Not Open)" {}
+   "Other (Open)" {}
+   "Other (Public Domain)" {}
+   "UK Open Government Licence (OGL v2)" {}
+   "UK Open Government Licence (OGL v3)" {}})
```

Comment:
Minimal. We can add things super easily to this, the empty maps are for links out to docs or other bits and bobs.

At some point we stick these all in a database and then people can add whatever they fancy.